src/sql/migrations.rs

```diff
@@ -497,6 +497,18 @@ item TEXT DEFAULT '');"#,
         )
         .await?;
     }
+    if dbversion < 81 {
+        info!(context, "[migration] v81");
+        sql.execute_migration(
+            r#"CREATE TABLE w30_status_updates (
+id INTEGER PRIMARY KEY AUTOINCREMENT,
+msg_id INTEGER,
+json TEXT DEFAULT '');
+CREATE INDEX w30_status_updates_index1 ON w30_status_updates (msg_id);"#,
+            81,
+        )
+        .await?;
+    }
```

Comment:
missing corresponding `tables.sql` update?

Reply:
apparently not, in the meantime i discovered `tables.sql` is out of date already.  i think it would be nicer to keep it in sync with the most recent db revision but that belongs in its own PR.